versioned/tests/src/main/java/org/projectnessie/versioned/tests/AbstractITVersionStore.java

```diff
@@ -904,6 +966,18 @@ protected void checkTransplantWithNoExpectedHash() throws VersionStoreException
               Optional.empty(),
               Optional.of("v4_1"),
               Optional.of("v5_1"));
+      List<WithHash<String>> commits = commitsList(newBranch);
+      store()
+          .getCommits(newBranch)
+          .map(w -> w.getValue())
+          .filter(m -> !m.equals("Another commit"))
+          .forEach(m -> assertMetaIsReset(m));
```

Comment:
I assume, this can be refactored into a better readable assertion w/o repetitive assertTrue/assertFalse